superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -397,6 +397,10 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
   alert = null,
   isReport = false,
 }) => {
+  const conf = useCommonConf();
+  const allowedNotificationMethods =
```

Comment:
should we cast this as `NotificationMethodOption[]` here so that we don't need to cast it multiple times below?